revsolver/base/openshift-templates/imageStream-template.yaml

```diff
@@ -0,0 +1,33 @@
+apiVersion: v1
```

Comment:
we can directly use the imagestream object in this file. 
As argocd `apply`'s the manifests, this would be template, whereas we need this as imagestream creation, right ?